src/math/Math.sol

```diff
@@ -6,12 +6,15 @@ pragma solidity ^0.8.0;
 /// @custom:contact security@morpho.xyz
 /// @dev Library to perform simple math manipulations.
 library Math {
-    function abs(int256 x) internal pure returns (int256 y) {
-        if (x == type(int256).min) return type(int256).max;
+    /* CONSTANTS */
+
+    // Only direct number constants and references to such constants are supported by inline assembly.
```

Comment:
```suggestion
    /// @dev Only direct number constants and references to such constants are supported by inline assembly.
```

Reply:
I'll open a PR on top to change this comment and discuss this